### ISSUE TITLE
Issue in bulk inserts with incomplete entries

### ISSUE THREAD

**Issue by matyas-igor** [koskimas/kysely#150]
Hi,

I'm having a problem with bulk inserts when entries have different sets of fields. In this case default value is forcefully being set to `null` which is not allowed.

Minimal example to reproduce:
```sql
create table test_inserts
(
  id           uuid                     not null default uuid_generate_v4(),
  field_text_a text                     not null default 'test1'::text,
  field_text_b text                     not null default 'test2'::text,
  created_at   timestamp with time zone not null default now(),
  primary key (id),
  check (field_text_a in ('test1', 'test2', 'test3')),
  check (field_text_b in ('test1', 'test2', 'test3'))
);
```
Important that here `field_text_a` and `field_text_b` both have default values.

And a TS code:
```ts
import { CamelCasePlugin, Generated, Kysely, PostgresDialect } from "kysely";
import { Pool } from "pg";

export interface TestInserts {
  id: Generated<string>;
  fieldTextA: Generated<"test1" | "test2" | "test3">;
  fieldTextB: Generated<"test1" | "test2" | "test3">;
  createdAt: Generated<Date>;
}

interface Database {
  testInserts: TestInserts;
}

const db = new Kysely<Database>({
  dialect: new PostgresDialect({
    pool: new Pool({
      host: "localhost",
      database: "<local_db_name>",
    }),
  }),
  plugins: [new CamelCasePlugin()],
});

async function main() {
  await db
    .insertInto("testInserts")
    .values([
      { fieldTextA: "test3" }, // here we only have field A !
      { fieldTextB: "test3" }, // here we only have field B !
    ])
    .execute();
}

main();
```

It seem that lib tries to execute the following query: 
```sql
insert into "test_inserts" ("field_text_a", "field_text_b") values ('test3', null), (null, 'test3');
```

Executing this gives an error:
```null value in column "field_text_b" of relation "test_inserts" violates not-null constraint```

Is there any way to overcome this problem with default values of columns?

I understand that it's some sort of an edge case, but in out previous setup with Knex/Objection we didn't have this problem in exactly the same code 🤔

**Comment by matyas-igor** [comment]
@koskimas hm, interesting, thanks for a solution! The real example is a lot more complicated, so probably not an option for me.. And I need to do the same for many tables and in a generic way. Will try to do it using a plugin

**Comment by koskimas** [comment]
We should probably use something other than `null` to fill in the blanks internally just like knex does. We could use `DEFAULT` or similar on all dialects that support it and then default to `null` on other dialects.

**Comment by koskimas** [comment]
Fixed in 0.21.6
